pincer/client.py

```diff
@@ -497,7 +498,7 @@ async def handle_middleware(
             raise RuntimeError(f"Middleware `{key}` has not been registered.")
 
         if next_call.startswith("on_"):
-            return next_call, ret_object
+            return (next_call, ret_object)
```

Comment:
uneeded?

Reply:
Ig they're required for pep8. I don't remember adding those lol.